components/raftstore/src/store/fsm/apply.rs

```diff
@@ -120,17 +120,17 @@ impl<C> PendingCmd<C> {
     }
 }
 
-impl<C> Drop for PendingCmd<C> {
-    fn drop(&mut self) {
-        if self.cb.is_some() {
-            safe_panic!(
-                "callback of pending command at [index: {}, term: {}] is leak",
-                self.index,
-                self.term
-            );
-        }
-    }
-}
+// impl<C> Drop for PendingCmd<C> {
```

Comment:
clean it?

Reply:
Fixed, thanks! （It was not cleaned up in the `ApplyFsm drop` before, which caused a panic when `stop_node` occurred, so I commented it out first.）